backend/gsr_booking/views.py

```diff
@@ -193,7 +191,7 @@ def get_queryset(self):
 
 class CheckWharton(APIView):
     def get(self, request):
-        return Response({"is_wharton": BW.is_wharton(request.user)})
+        return Response({"is_wharton": WhartonGSRBooker.is_wharton(request.user)})
```

Comment:
Let's add a check to see if `request.user` is not None first and default return false if that is the case. That way we can still let not-logged in users to hit this route.

Reply:
hm not sure if this will ever be `None` since think the default is `AnonymousUser`. any reason we don't wanna just make the route Authenticated?

Reply:
Oh I was thinking we shouldn't make this Authenticated because I remember on iOS you could still view GSRs even when not logged in? But thinking about it some more I think making this authenticated is a good idea